sbin/build.sh

```diff
@@ -442,16 +442,28 @@ getGradleHome() {
   echo $gradleJavaHome
 }
 
+getGradleUserHome() {
+  local gradleUserHome=""
+
+  if [ -z "${BUILD_CONFIG[GRADLE_USER_HOME]}" ]; then
+    gradleUserHome="${BUILD_CONFIG[WORKSPACE_DIR]}/.gradle"
+  else
+    gradleUserHome="${BUILD_CONFIG[GRADLE_USER_HOME]}"
+  fi
+
+  echo $gradleUserHome
```

Comment:
make the echo more verbose so there's context

Reply:
I thought this serves as a return statement -- is this not how you return values in bash?

Reply:
Ah, I see what you're doing now, I'm being dense - carry on :-)